src/Compilers/Test/Core/TargetFrameworkUtil.cs

```diff
@@ -113,13 +113,26 @@ public static class NetCoreApp
     /// </summary>
     public static class NetFramework
     {
+        private static ImmutableArray<MetadataReference> s_references;
+
         /// <summary>
         /// This is the full set of references provided by default on the .NET Framework TFM
         /// </summary>
         /// <remarks>
         /// Need to special case tuples until we move to net472
         /// </remarks>
-        public static ImmutableArray<MetadataReference> References { get; } = [.. Net461.References.All, Net461.ExtraReferences.SystemValueTuple];
+        public static ImmutableArray<MetadataReference> References
```

Comment:
There are a number of tests that call say `NetFramework.mscorlib`. When this was a static initializer that realized all these values when they didn't really need to be. This is only a savings in certain silces of our tests but it's a significant one in them.